app/components/agent/Show.js

```diff
@@ -445,6 +445,14 @@ class AgentShow extends React.Component {
   }
 }
 
+/*
+Disabling graphql/no-deprecated-fields here as I think there is some changes probably
+required in the graph implementation so that we can actually use the alternatives to
+`pingedAt`, `stoppedAt`, & `stoppedBy` fields as at the moment it seems like using these
+fields will mean we need do a lot of extra fetching and checking on the client which
+seems bad?
```

Comment:
Thoughts welcome. This seems to me like the pragmatic way to get this shipped and catching new errors.